runtime/vm/vmthread.c

```diff
@@ -1392,6 +1392,7 @@ allocateJavaStack(J9JavaVM * vm, UDATA stackSize, J9JavaStack * previousStack)
 		stack->size = stackSize;
 		stack->previous = previousStack;
 		stack->firstReferenceFrame = 0;
+		memset((void*) ( (uintptr_t) stack + sizeof(J9JavaStack)), 0, mallocSize - sizeof(J9JavaStack));
```

Comment:
The bytecode verifier already validates that bytecodes can't access uninitialized memory on the Java stack.  memseting the stack is an unnecessary runtime overhead.